spark/src/test/scala/org/apache/spark/sql/delta/SchemaValidationSuite.scala

```diff
@@ -343,7 +337,7 @@ class SchemaValidationSuite extends SchemaValidationSuiteBase {
     actionToTest = (spark: SparkSession, tblPath: String) => {
       val deltaTable = io.delta.tables.DeltaTable.forPath(spark, tblPath)
       val sourceDf = spark.range(10).withColumn("col2", lit(2))
-      val e = intercept[Exception] {
+      val e = intercept[DeltaAnalysisException] {
```

Comment:
Can we update the comment above, `throws an DeltaAnalysisException`.

Reply:
Done